specification/riskiq/Easm/models.tsp

```diff
@@ -270,6 +323,42 @@ model AssetUpdateData {
   transfers?: AssetUpdateTransfers;
 }
 
+@doc("A request body used to export an asset.")
+@Versioning.added(Versions.v2024_03_01_preview)
+model AssetsExportRequestBody {
+  @doc("The name of the file to export.")
+  fileName: string;
+
+  @doc("The columns to export.")
+  columns: string[];
+}
+
+@doc("A request body used to retrieve a list of deltas.")
+@Versioning.added(Versions.v2024_03_01_preview)
+model DeltaDetailsRequestBody {
+  @doc("The type of delta detail to retrieve.")
+  deltaDetailType?: DeltaDetailType;
+
+  @doc("The number of days prior to retrieve deltas for.")
+  daysPrior?: int32;
+
+  @doc("The type of asset")
+  kind?: GlobalAssetType;
+
+  @doc("expected format to be: yyyy-MM-dd")
+  date?: string;
+}
+
+@doc("A request body used to retrieve a delta summary.")
+@Versioning.added(Versions.v2024_03_01_preview)
+model DeltaSummaryRequestBody {
```

Comment:
Let's remove the "body" suffix. Is either value required? What happens if you provide neither?

Reply:
removed and also marked value as required